config/production.json

```diff
@@ -37,7 +37,7 @@
 		"happychat": true,
 		"help": true,
 		"help/courses": true,
-		"hosting": false,
+		"hosting": true,
```

Comment:
For other reviewers, note that'd we need to also whitelist the new route for WordPress.com before this is actually visible

Reply:
Whitelisted. 👍 thanks for the reminder.